composer.lock

```diff
@@ -289,7 +289,7 @@
     "packages-dev": [
         {
             "name": "ampproject/php-css-parser-install-plugin",
-            "version": "dev-develop",
+            "version": "dev-remove/sandboxing-filter-opt-in",
```

Comment:
This line, changed by the `ergebnis/composer-normalize`, looked a bit suspicious to me. However, I did a fresh install of the plugin from the repo and everything seems to be working fine.